### PR TITLE
Update contributors list #601

### PR DESCRIPTION
**Pull request by ake123**
Update contributors list #601

### CODE REVIEW

File: oma_packages/oma_packages.csv [hunk 1]
```diff
@@ -75,3 +75,4 @@ tidyverse
 topGO
 vegan
 WGCNA
+NMF
```
Comment (TuomasBorman) [inline oma_packages/oma_packages.csv]: Packages should be in alphabetical order

**Review by TuomasBorman — CHANGES_REQUESTED** [review]

File: oma_packages/oma_packages.csv (hunk 1 above)
Comment (ake123) [reply oma_packages/oma_packages.csv]: fixed

**Comment by antagomir** [comment]
resolve conflicts

**Review by TuomasBorman — APPROVED** [review]

**Comment by TuomasBorman** [comment]
Can you revert this? Look the book https://microbiome.github.io/OMA/docs/devel/ This is too long list.

Instead, can you update the list in here https://microbiome.github.io/OMA/docs/devel/pages/acknowledgments.html

**Comment by ake123** [comment]
Yes sure

**Pull request merged**